src/main/java/stringcalculator/StringAddCalculator.java

```diff
@@ -0,0 +1,81 @@
+package stringcalculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\n(.*)");
+
+    public static int splitAndAdd(String input) {
+        if (input == null || input.isEmpty()) {
+            return 0;
+        }
+
+        return add(split(parseInputExpression(input)));
+    }
+
+    private static int add(String[] stringNumberArray) {
+        return Arrays.stream(stringNumberArray).mapToInt(StringAddCalculator::parseInt).sum();
+    }
```

Comment:
연산마다 개행을 이용하면 가독성을 높일 수 있을 것 같아요 :)

```suggestion
    private static int add(String[] stringNumberArray) {
        return Arrays.stream(stringNumberArray)
            .mapToInt(StringAddCalculator::parseInt)
            .sum();
    }
```